rastervision/utils/files.py

```diff
@@ -1,8 +1,10 @@
 import os
 import shutil
-from threading import Timer
+import time
 import logging
 
+from threading import Timer
```

Comment:
nitpick - We've been keeping the structure as

```python
[library imports]

[3rd party imports]

[rastervision imports]
```

Is there a reason to add the extra section? If not we should keep to that convention, as a tidiness thing.

Reply:
Ah, I see.  I had not recognized the pattern.